### DISCUSSION TITLE
I would like to translate revolt to georgian

### DISCUSSION THREAD

**Discussion by silacs** [stoatchat/discussions#404] [Q&A] [answered]
the title

**Comment by Rexogamer** [comment, marked as answer]
Hey! We appreciate your interest in helping us to translate Revolt. You can do so via our [Weblate instance](https://weblate.insrt.uk) - if you have any questions on how to use Weblate or what specific strings are used for, feel free to ask us in the i18n channel in the [Revolt Lounge](https://rvlt.gg/Testers). Thanks!

**Comment by silacs** [comment]
damn seems like i forgot to login to github after I posted this...... @Rexogamer
probably will translate if I have the time
